libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -127,9 +127,12 @@ static volatile EventGroupHandle_t xSyncEventGroup = NULL;
 /* Bit definitions used with the xSyncEventGroup event group to allow the
  * prvEchoClientTxTask() and prvEchoClientRxTask() tasks to synchronize before
  * commencing a new cycle with a different socket. */
-#define tcptestTX_TASK_BIT    ( 0x01 << 1 )
-#define tcptestRX_TASK_BIT    ( 0x01 << 2 )
+#define tcptestTX_TASK_BIT                                  ( 0x01 << 1 )
+#define tcptestRX_TASK_BIT                                  ( 0x01 << 2 )
 
+#ifndef Threadsafe_SameSocketDifferentTasksFrameSize
+    #define Threadsafe_SameSocketDifferentTasksFrameSize    ( 2 * 1500 )
```

Comment:
It looks like this can be defined from `tcptestTWICE_MAX_FRAME_SIZE`?
```
#define Threadsafe_SameSocketDifferentTasksFrameSize tcptestTWICE_MAX_FRAME_SIZE
```

Reply:
Also, the naming convention is a bit different. How come we are no longer prefixing with `tcptest`?

Reply:
Thanks for the suggestion, fixed in 6765b43